data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,85 @@
+---
+title: New Electron Release Cadence
+author: VerteDinde
+date: '2021-07-14'
+---
+
+Beginning in September 2021, Electron will release a new major stable version every 8 weeks.
+
+---
+
+In 2019, Electron moved to a 12 week release cycle to match Chromium's 6 week release cycle. Recently, both Chrome and Microsoft announced changes that made us reconsider Electron's current release cadence:
+
+1. Chromium plans to [release a new milestone every **4 weeks**, starting with Chrome 94 on September 21st, 2021.](https://blog.chromium.org/2021/03/speeding-up-release-cycle.html) This release cadence also adds a new Extended Stable option every 8 weeks, which will contain all updated security fixes.
+
+2. The Microsoft Store will [require Chromium-based apps to be no older than within 2 major versions](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#102-security) (for example, if the latest released major version of Chromium is 85, any browser based on Chromium must be on at least Chromium version 83 or higher). This rule includes Electron apps.
```

Comment:
```suggestion
2. The Microsoft Store will [require Chromium-based apps to be no older than within 2 major versions](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#102-security). As an example, if the latest released major version of Chromium is 85, any browser based on Chromium must be on at least Chromium version 83 or higher. **This rule includes Electron apps.**
```

Reply:
Did this because it's nice to not have entire ideas that can stand on their own in parentheticals. Also, it's nice to boldface that Electron apps are incldued imo.